### DISCUSSION TITLE
代码高亮的css 和 hexo-theme-yun.css 好像冲突了，该怎么办

### DISCUSSION THREAD

**Discussion by wgx-dh** [YunYouJun/hexo-theme-starry#308] [Q&A 疑难杂症] [answered]
![image](https://user-images.githubusercontent.com/63238251/170927307-8921ac13-1f98-4ffa-ac2b-74e62bffd95e.png)

**主题版本 1.9.1**

**代码高亮用的 prism-okaidia.css**

**关于 char 的 css 有冲突**

**Comment by YunYouJun** [comment]
I will fixed it asap.

**Comment by YunYouJun** [comment, marked as answer]
Fixed and released in `v1.10.3`.
